developers/weaviate/search/bm25.md

```diff
@@ -324,6 +361,15 @@ import TknTsCodeLegacy from '!!raw-loader!/_includes/code/howto/manage-data.coll
     />
   </TabItem>
 
+  <TabItem value="go" label="Go">
```

Comment:
@donomii I think you said in the channel that it would take a couple of days to get the tokenization code. 

Let's remove this block in the meantime.

Reply:
I removed it in this [PR](https://github.com/weaviate/weaviate-io/pull/2462)